howto.md

```diff
@@ -19,9 +47,10 @@ permalink: howto.html
       </div>
    
 5. Make changes to your code using your IDE.  Save and refresh the page to see the changes locally.
-6. Once happy, commit your code and publish your branch
-7. On Github, make a new `Pull requests`
-8. Ask another team member to review the PR and merged into the live website!
+6. If required, submit content to internal gateway and await content review
+7. Once happy, commit your code and publish your branch
+8. On Github, make a new `Pull requests`
+9. Ask another team member to review the PR and merged into the live website!
 
 **Alternatively**, for small changes such as the addtions listed below, you can ammend the ymls and add assets/markdowns directly on github but please do this within a branch and pull rather than direct to the main.
```

Comment:
ymls -> `.ymls`
and pull rather than direct to the main -> and pull request rather than directly on to the main